### PR TITLE
EDR SWG telecom number 11 changes

### PR DESCRIPTION
**Pull request by m-burgoyne**
Removed the **resolution** query parameter from `/trajectory` and `/corridor`.
Added a **z** query parameter to `/position` and `/radius`.
Set the published date back to  yyyy-mm-dd
Added a link to the OpenAPI definitions in the examples Appendix.

### CODE REVIEW

**Comment by pavol-novotny** [comment]
https://github.com/opengeospatial/Environmental-Data-Retrieval-API/issues/53#issuecomment-628845428
> The optional resolution is optional for the Core API.

https://github.com/opengeospatial/Environmental-Data-Retrieval-API/pull/64#issue-418496461
> Removed the resolution query parameter from /trajectory and /corridor.

@m-burgoyne @chris-little I am a little bit confused. Is the `resolution` parameter optional for the `/trajectory` query or has it been completely removed from the Core API?

File: candidate-standard/openapi/EDR_OpenAPI.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "openapi": "3.0.2",
   "info": {
-    "title": "Environmental Data Retreival API's",
-    "description": "Example of Candidate Environmental Data Retreival API OpenAPI docs",
-    "version": "0.0.5"
+    "title": "Environmental Data Retrieval API's",
+    "description": "Example of Candidate Environmental Data Retrieval API OpenAPI docs",
+    "version": "0.0.6"
```
Comment (dblodgett-usgs) [inline candidate-standard/openapi/EDR_OpenAPI.json:6]: Should the version here be one ahead of the version of the spec? in: candidate-standard/EDR-candidate-specification.adoc ?

Comment (m-burgoyne) [reply candidate-standard/openapi/EDR_OpenAPI.json:6]: @dblodgett-usgs No the two version values should be synchronised

**Comment by m-burgoyne** [comment]
@pavol-novotny Yes resolution has been removed from `/trajectory` and `/corridor`, the user will have to specify the coordinates that they require values for using the **coords** query parameter.

**Comment by pavol-novotny** [comment]
> ... the user will have to specify the coordinates that they require values for using the **coords** query parameter.

@m-burgoyne I suppose that this is related just to the `/trajectory` query. In contrast, in a case of the `/corridor` query, all the data within the specified volume should be returned. So, in a case of gridded data, it is not expected that the axes of the returned grid will be aligned along the corridor's geometry (obtained by interpolation of original data). Instead, all grid points from the original grid lying within the specified corridor volume should be returned. Correct?

**Comment by m-burgoyne** [comment]
@pavol-novotny In the case of the `/corridor` API the **optional** **resolutionx** query parameter for interpolating along the corridor_width and **resolutionz**  query parameter for interpolating along the corridor_height  are still available, so the collection provider can decide how to align the resulting data if interpolation is appropriate and feasible.
Removing the `resolution` parameter still allows for interpolation along the path (if the collection publisher supports it) but the intervals are defined by the user input coordinates.

EDR endpoint support for interpolation of the data is not prescribed by the API but left to the collection publisher to decide if they will implement the capability.  If the collection does not support interpolation, then the data will be returned on the original grid.

File: candidate-standard/openapi/EDR_OpenAPI.json (hunk 1 above)
Comment (dblodgett-usgs) [reply candidate-standard/openapi/EDR_OpenAPI.json:6]: 👍

**Comment by dblodgett-usgs** [comment]
Point of order, when merging this issue, please link it to the appropriate issue with the `#{0-9}{0-9}` syntax.

**Comment by dblodgett-usgs** [comment]
@pavol-novotny Can you please "opt-in" to the git repository in the OGC Portal here: https://portal.ogc.org/?m=projects&a=view&project_id=627 so I can tag you as a reviewer of this pull request?

**Comment by pavol-novotny** [comment]
> @pavol-novotny Can you please "opt-in" to the git repository in the OGC Portal here: https://portal.ogc.org/?m=projects&a=view&project_id=627 so I can tag you as a reviewer of this pull request?

@dblodgett-usgs I am not sure what you mean by "opt-in" to the git repository in the OGC Portal - sorry, I am quite new at OGC. I can see myself as a *Group Member* in the list of contacts for the *Environmental Data Retrieval API SWG* project. I have also created an account in the OGC GitLab, but I cannot find any project visible for me. Is there anything else I should do?

**Comment by dblodgett-usgs** [comment]
Sorry... this shouldn't be hard and isn't really THAT important. If you can check that your github ID is registered in your OGC Portal user then go back to the project, you should see an "Opt In to github" button. 

If not, once you submit and we merge a pull request, I think github will see you as a contributor and allow me to tag you so we can get it fixed in the long run.

**Comment by pavol-novotny** [comment]
> Sorry... this shouldn't be hard and isn't really THAT important. If you can check that your github ID is registered in your OGC Portal user then go back to the project, you should see an "Opt In to github" button.

@dblodgett-usgs I can see my `GithubID` (`pavol-novotny`) in the OGC Portal:

OGC_profile

However, I cannot see any "Opt In to github" button on the project page:

OGC_EDR_project

**Comment by ogcscotts** [comment]
@dblodgett-usgs @pavol-novotny This is a public repo, so there will not be an "opt in" feature on the OGC Portal.
Dave - you should be able to invite Pavol to be a collaborator.

**Comment by dblodgett-usgs** [comment]
Oh, my mistake. I thought that was how people became members of the group associated with the repository.

**Comment by dblodgett-usgs** [comment]
@ogcscotts how do I get @pavol-novotny into: https://github.com/orgs/opengeospatial/people ?

**Comment by ogcscotts** [comment]
@dblodgett-usgs Of course, nothing is ever straightforward: closed repos do require the Opt In. I am taking an action to give a little more guidance on our Gits.

**Comment by ogcscotts** [comment]
@dblodgett-usgs with respect to adding @pavol-novotny - I will get that done (not sure how the magic happens, but will ask the magician)

**Pull request merged**